src/lib/import_export/csv_parser.cpp

```diff
@@ -84,7 +84,9 @@ std::shared_ptr<Table> CsvParser::parse(const std::string& filename, const std::
   CurrentScheduler::wait_for_tasks(tasks);
 
   for (auto& segments : segments_by_chunks) {
-    table->append_chunk(segments);
+    DebugAssert(!segments.empty(), "Empty chunks shouldn't occur when importing CSV");
```

Comment:
What happens if I import an empty CSV?

Reply:
This only checks whether a `Chunk` has any columns. So with this it is not possible to import a table without columns.

Reply:
`TEST_F(OperatorsImportCsvTest, NoRows)` verifies that it is valid to import a CSV without rows - it just won't have any chunks.